empress/recon_vis/utils.py

```diff
@@ -3,17 +3,75 @@
 Utilities related to conversion between data types
 """
 
-from collections import OrderedDict 
 from typing import Dict, Tuple, List
-from empress.recon_vis.recon import MappingNode, Reconciliation
+from empress.recon_vis.recon import MappingNode, ReconGraph, Reconciliation
 from empress.recon_vis.recon import Cospeciation, Duplication, Transfer, Loss, TipTip
-from empress.recon_vis import tree
+import empress.recon_vis.tree as tree
 from empress.recon_vis.tree import NodeLayout
 from empress.recon_vis.tree import TreeType
+from collections import OrderedDict 
 from enum import Enum
 
 __all__ = ['dict_to_tree', 'dict_to_reconciliation', 'build_trees_with_temporal_order']
 
+# Master utility function coverts from dictionaries to objects
+
+def convert_to_objects(host_dict, parasite_dict, recon_dict, event_frequencies):
```

Comment:
I feel like I have seen this code somewhere else. Is this a duplicate code?

Reply:
I'm not sure. We didn't have much to do with utils.py, as it was given to us

Reply:
fixed